lib/class-wp-theme-json-gutenberg.php

```diff
@@ -956,6 +956,27 @@ public function get_settings() {
 		}
 	}
 
+	/**
+	 * Processes the CSS, to apply nesting.
+	 *
+	 * @param string $css      The CSS to process.
+	 * @param string $selector The selector to nest.
+	 *
+	 * @return string The processed CSS.
+	 */
+	public function process_blocks_custom_css_nested_styles( $css, $selector ) {
```

Comment:
Sorry for nitpicking here. The "nested styles" is the result of processing, right? In that case, would it make sense to call this method `process_blocks_custom_css`?

Reply:
makes sense... renamed and committed 👍